account/src/main/java/com/user/account/controller/AllControlers.java

```diff
@@ -28,38 +49,56 @@ public String home() {
 		// Updating accountBalance after debit
 		
 		
-		@PostMapping("/debit/{accountNumber}/{amount}")
-		public User debit(@PathVariable String accountNumber, @PathVariable String amount) {
+		@PostMapping("/debit/{accountNumber}")
+		public void debit(@PathVariable String accountNumber, @RequestBody String amount) {
+
+			long userAccountNumber = Long.parseLong(accountNumber);
+
+			long debitAmount = Long.parseLong(amount);
```

Comment:
What if amount is `non-numeric`?

Shouldn't you show the appropriate message to the user?

Reply:
?